Source/WebKit/WPEPlatform/wpe/wayland/WPEViewWayland.cpp

```diff
@@ -83,6 +83,29 @@ struct DMABufFeedback {
         {
         }
 
+        explicit FormatTable() = default;
+        explicit FormatTable(uint32_t size, int fd)
+            : size(size)
+            , data(static_cast<FormatTable::Data*>(mmap(nullptr, size, PROT_READ, MAP_PRIVATE, fd, 0)))
+        {
+            if (data == MAP_FAILED) {
+                data = nullptr;
+                size = 0;
+            }
+        }
+
+        explicit operator bool() const
+        {
+            return size && data;
+        }
+
+        FormatTable& operator=(FormatTable&& other)
+        {
+            std::swap(size, other.size);
+            std::swap(data, other.data);
+            return *this;
+        }
```

Comment:
This is inconsistent with the move constructor. We can change the move constructor to just do: `*this = WTFMove(other);` so that if this changes in the future we only need to change the operator here.

Reply:
Good idea, I'll apply the simplification.